### DISCUSSION TITLE
Make Dice So Nice recognize all rolls?

### DISCUSSION THREAD

**Discussion by tbergstedt** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#163] [Q&A] [answered]
I noticed that when adding special effects on dice results in Dice So Nice, "Crosses" aren't recognized as sixes, and "Skulls" not seen as one's. Would be nice it they could.

**Comment by aMediocreDad** [comment, marked as answer]
Hey! I am not sure I understand correctly what you are requesting. Dice So Nice Special Effects work as expected with the proprietary Forbidden Lands Dice (db, ds, dg and dn). 

See here: https://streamable.com/3pcu7d

The settings I used for this is here:
![image](https://user-images.githubusercontent.com/9851733/132977978-88d46abe-cb18-45c6-a959-1069f2d2404b.png)

Is this what you meant with your request?

**Reply by tbergstedt** [reply]
I get it now - didn't realize they were of a specific dice category. I only had effects set for the "d6" type, not "db" and the others. Thanks for pointing that out.

**Reply by aMediocreDad** [reply]
No problem! Forbidden Lands needs to use its own dice types in order to avoid messing with normal d6 rolls (that don't count successes). The benefit is that this makes the special effects really customizable (like not having an effect on 1's rolled on Skill/Negative dice).
